images/Dockerfile.alpine-arm-rpi

```diff
@@ -82,9 +82,7 @@ RUN apk --no-cache add  \
       open-vm-tools-deploypkg \
       open-vm-tools-guestinfo \
       open-vm-tools-static \
-      open-vm-tools-vmbackup \
-      zfs \
-      zfs-lts
```

Comment:
in here they only mention zfs package: https://wiki.alpinelinux.org/wiki/Root_on_ZFS_with_native_encryption

Reply:
I'm not sure, @paynejacob had a look at this - is it used?

Reply:
I'm suspecting there is a toolset and kernel modules packages probably

Reply:
but on the other hand... zfs on RPI? I don't think that's widely used on RPI/ARM given the resource constraints, but if someone misses it can open an issue and we can have a look at it again

Reply:
right, its kernel modules.  my bet this is not working at all on RPIs as we have non-musl kernels and userland is musl. pretty bad, this is another reason for #653